packages/contractkit/src/wrappers/SortedOracles.ts

```diff
@@ -86,43 +87,38 @@ export class SortedOraclesWrapper extends BaseWrapper<SortedOracles> {
   /**
    * Updates an oracle value and the median.
    * @param token The address of the token for which the Celo Gold exchange rate is being reported.
-   * @param numerator The amount of tokens equal to `denominator` Celo Gold.
-   * @param denominator The amount of Celo Gold that the `numerator` tokens are equal to.
+   * @param value The number of `token`s equal to one Celo Gold.
    */
   async report(
     token: CeloToken,
-    numerator: number,
-    denominator: number,
+    value: BigNumber.Value,
     oracleAddress: Address
   ): Promise<CeloTransactionObject<void>> {
     const tokenAddress = await this.kit.registry.addressFor(token)
+    const fixedValue = toFixed(valueToBigNumber(value))
 
     const { lesserKey, greaterKey } = await this.findLesserAndGreaterKeys(
       token,
-      numerator,
-      denominator,
+      valueToBigNumber(value),
```

Comment:
Shouldn't this be `toFixed`ed?

Reply:
Not needed as `findLesserAndGreaterKeys` compares this with the existing rates, which are `fromFix`ed